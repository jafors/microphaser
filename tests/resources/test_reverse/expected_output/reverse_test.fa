>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>0b4e37d09934bdeR
acAgggacAggggccgggaccgggacc
>0bf5651d15c846cR
acAgggactggggccgggaccgggaca
>5829bfd284f0cb5R
gggacAgggacAggggccgggaccggg
>683eeee967ad8d0R
gggacAgggactggggccgggaccggg
>4087657e230f3a5R
accgggacAgggacAggggccgggacc
>7023ad3880e2d57R
accgggacAgggactggggccgggacc
>8c3434d759140efR
gggaccgggacAgggactggggccggg
>ae280e232d0582aR
gggaccgggacAgggacAggggccggg
>6f5b9cce754f8f2R
gggaccgggacAgggactggggccggg
>561f80087272f47R
gccgggaccgggacAgggactggggcc
>e1b098d6a1588e2R
gccgggaccgggacAgggacAggggcc
>e9da669b9cab341R
ggAgccgggaccgggacAgggactggg
>4e047277cdeb322R
ggggccgggaccgggacAgggacAggg
>fc553bd6250d877R
actggAgccgggaccgggacAgggact
>f561469dc0d1bbdR
actggggccgggaccgggacAgggacA
>8aaf4c7d467b4f4R
gggactggggccgggaccgggacAggg
>cf58ccde40a93c6R
gggactggAgccgggaccgggacAggg
>18fa98f4b970791R
accgggactggggccgggaccgggacA
>f7a0e246701c5ddR
acAgggactggggccgggaccgggacA
>d6d3ec26afc54dbR
acAgggactggAgccgggaccgggacA
