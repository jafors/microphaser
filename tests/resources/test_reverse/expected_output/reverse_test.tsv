id	transcript	gene_id	gene_name	chrom	offset	freq	depth	nvar	nsomatic	nvariant_sites	nsomvariant_sites	strand	somatic_positions	somatic_aa_change	germline_positions	germline_aa_change	normal_sequence	mutant_sequence
a324f17d17b218dR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300986	0.5327102803738317	107	1	1	1	1	Reverse	26300986	p.Ser46Ser			TGAGCCACACCCATCACTCAACATGTC	cGAGCCACACCCATCACTCAACATGTC
c258647e023393eR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300983	0.5137614678899083	109	1	1	1	1	Reverse	26300986	p.Ser46Ser			TTCTGAGCCACACCCATCACTCAACAT	TTCcGAGCCACACCCATCACTCAACAT
172c41a2e6e2cb7R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300980	0.5087719298245614	114	1	1	1	1	Reverse	26300986	p.Ser46Ser			TTCTTCTGAGCCACACCCATCACTCAA	TTCTTCcGAGCCACACCCATCACTCAA
b5f424dc651bf48R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300977	0.5042016806722689	119	1	1	1	1	Reverse	26300986	p.Ser46Ser			CTTTTCTTCTGAGCCACACCCATCACT	CTTTTCTTCcGAGCCACACCCATCACT
965e45175515136R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300974	0.5040650406504065	123	1	1	1	1	Reverse	26300986	p.Ser46Ser			GATCTTTTCTTCTGAGCCACACCCATC	GATCTTTTCTTCcGAGCCACACCCATC
08add1803a50f62R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300971	0.4838709677419355	124	1	1	1	1	Reverse	26300986	p.Ser46Ser			TGAGATCTTTTCTTCTGAGCCACACCC	TGAGATCTTTTCTTCcGAGCCACACCC
223bd8fb09288ccR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300968	0.4793388429752066	121	1	1	1	1	Reverse	26300986	p.Ser46Ser			GACTGAGATCTTTTCTTCTGAGCCACA	GACTGAGATCTTTTCTTCcGAGCCACA
732c0f3ff28e5c0R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300965	0.48760330578512395	121	1	1	1	1	Reverse	26300986	p.Ser46Ser			AGGGACTGAGATCTTTTCTTCTGAGCC	AGGGACTGAGATCTTTTCTTCcGAGCC
58c2ffd43739c73R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26300962	0.5245901639344263	122	1	1	1	1	Reverse	26300986	p.Ser46Ser			GGAAGGGACTGAGATCTTTTCTTCTGA	GGAAGGGACTGAGATCTTTTCTTCcGA
742163ac1cebc12R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284398	1	32	1	1	1	1	Reverse	26284399	p.Leu312Arg			CAGCTGCCTGCCCACCACACGGCCCTC	CcGCTGCCTGCCCACCACACGGCCCTC
fd764a5345f7473R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284395	1	33	1	1	1	1	Reverse	26284399	p.Leu312Arg			CATCAGCTGCCTGCCCACCACACGGCC	CATCcGCTGCCTGCCCACCACACGGCC
a55d1f2587eae5cR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284392	1	34	1	1	1	1	Reverse	26284399	p.Leu312Arg			GTGCATCAGCTGCCTGCCCACCACACG	GTGCATCcGCTGCCTGCCCACCACACG
ae580a513507433R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284389	1	33	1	1	1	1	Reverse	26284399	p.Leu312Arg			CTTGTGCATCAGCTGCCTGCCCACCAC	CTTGTGCATCcGCTGCCTGCCCACCAC
946de2980abc10fR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284386	1	33	1	1	1	1	Reverse	26284399	p.Leu312Arg			GGCCTTGTGCATCAGCTGCCTGCCCAC	GGCCTTGTGCATCcGCTGCCTGCCCAC
4d2f7903355cae3R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284383	1	32	1	1	1	1	Reverse	26284399	p.Leu312Arg			CAAGGCCTTGTGCATCAGCTGCCTGCC	CAAGGCCTTGTGCATCcGCTGCCTGCC
8a68b9eec214f8bR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284380	1	33	1	1	1	1	Reverse	26284399	p.Leu312Arg			GTCCAAGGCCTTGTGCATCAGCTGCCT	GTCCAAGGCCTTGTGCATCcGCTGCCT
56239f5beba4a30R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284377	1	33	1	1	1	1	Reverse	26284399	p.Leu312Arg			CCTGTCCAAGGCCTTGTGCATCAGCTG	CCTGTCCAAGGCCTTGTGCATCcGCTG
c8cd1f33758a3fcR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26284374	1	35	1	1	1	1	Reverse	26284399	p.Leu312Arg			CACCCTGTCCAAGGCCTTGTGCATCAG	CACCCTGTCCAAGGCCTTGTGCATCcG
0b4e37d09934bdeR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282373	0.034482758620689655	29	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282381	p.Ser494Cys	accgggacAggggccgggaccgggacc	acAgggacAggggccgggaccgggacc
0bf5651d15c846cR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282373	0.13793103448275862	29	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	accgggactggggccgggaccgggaca	acAgggactggggccgggaccgggaca
5829bfd284f0cb5R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282370	0.03571428571428571	28	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282381	p.Ser494Cys	gggaccgggacAggggccgggaccggg	gggacAgggacAggggccgggaccggg
683eeee967ad8d0R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282370	0.14285714285714285	28	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	gggaccgggactggggccgggaccggg	gggacAgggactggggccgggaccggg
4087657e230f3a5R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282367	0.04	25	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282381	p.Ser494Cys	accgggaccgggacAggggccgggacc	accgggacAgggacAggggccgggacc
7023ad3880e2d57R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282367	0.16	25	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	accgggaccgggactggggccgggacc	accgggacAgggactggggccgggacc
8c3434d759140efR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282364	0.09090909090909091	22	1	1	3	1	Reverse	26282375	p.Gly496Cys			gggaccgggaccgggactggggccggg	gggaccgggacAgggactggggccggg
ae280e232d0582aR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282364	0.045454545454545456	22	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282381	p.Ser494Cys	gggaccgggaccgggacAggggccggg	gggaccgggacAgggacAggggccggg
6f5b9cce754f8f2R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282364	0.13636363636363635	22	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282385	p.Pro491_Gly492del	gggaccgggaccgggactggggccggg	gggaccgggacAgggactggggccggg
561f80087272f47R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282361	0.13333333333333333	15	1	1	3	1	Reverse	26282375	p.Gly496Cys			gccgggaccgggaccgggactggggcc	gccgggaccgggacAgggactggggcc
e1b098d6a1588e2R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282361	0.06666666666666667	15	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282381	p.Ser494Cys	gccgggaccgggaccgggacAggggcc	gccgggaccgggacAgggacAggggcc
e9da669b9cab341R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282358	0.1875	16	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282360	p.Pro501Ser	ggAgccgggaccgggaccgggactggg	ggAgccgggaccgggacAgggactggg
4e047277cdeb322R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282358	0.0625	16	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282381	p.Ser494Cys	ggggccgggaccgggaccgggacAggg	ggggccgggaccgggacAgggacAggg
fc553bd6250d877R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282355	0.2	15	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282360	p.Pro501Ser	actggAgccgggaccgggaccgggact	actggAgccgggaccgggacAgggact
f561469dc0d1bbdR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282355	0.06666666666666667	15	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282381	p.Ser494Cys	actggggccgggaccgggaccgggacA	actggggccgggaccgggacAgggacA
8aaf4c7d467b4f4R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282352	0.15384615384615385	13	1	1	2	1	Reverse	26282375	p.Gly496Cys			gggactggggccgggaccgggaccggg	gggactggggccgggaccgggacAggg
cf58ccde40a93c6R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282352	0.23076923076923078	13	2	1	2	1	Reverse	26282375	p.Gly496Cys	26282360	p.Pro501Ser	gggactggAgccgggaccgggaccggg	gggactggAgccgggaccgggacAggg
18fa98f4b970791R	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282349	0.07692307692307693	13	1	1	3	1	Reverse	26282375	p.Gly496Cys			accgggactggggccgggaccgggacc	accgggactggggccgggaccgggacA
f7a0e246701c5ddR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282349	0.07692307692307693	13	2	1	3	1	Reverse	26282375	p.Gly496Cys	26282351	p.Gly504Cys	acAgggactggggccgggaccgggacc	acAgggactggggccgggaccgggacA
d6d3ec26afc54dbR	ENST00000314675	ENSG00000158062	UBXN11	chr1	26282349	0.23076923076923078	13	3	1	3	1	Reverse	26282375	p.Gly496Cys	26282351|26282360	p.Gly504Cys|p.Pro501Ser	acAgggactggAgccgggaccgggacc	acAgggactggAgccgggaccgggacA
